### DISCUSSION TITLE
How can I check and extract distillation part of soft teacher model?

### DISCUSSION THREAD

**Discussion by KJB00** [community/community#70549] [Programming Help]
### Select Topic Area

Question

### Body

I am studying about distillation. 

To learn more about this, I would like to check the distillation code of the Soft Teacher method and extract it. But I can't find distillation part in Soft Teacher code.

Is there a way anybody can help me?

**Comment by Xrenya** [comment]
@KJB00 it was trained using MMDetection and the distillation part in located [here](https://github.com/microsoft/SoftTeacher/blob/main/ssod/models/soft_teacher.py).
